deployment/deployment.json

```diff
@@ -1220,8 +1220,8 @@
         "status": "prod",
         "versions": {
           "schema": "2.0.1",
-          "subgraph": "1.1.9",
-          "methodology": "1.0.0"
+          "subgraph": "1.2.0",
```

Comment:
I would say this is major, what do you think? A critical change

Reply:
Either seems fine to me. Since the realms are all much smaller, one may see them as auxiliary and adding them is "minor"?